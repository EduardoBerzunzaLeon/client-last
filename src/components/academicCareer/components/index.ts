export * from './StudentCard';
export * from './UserCreatorCard';
