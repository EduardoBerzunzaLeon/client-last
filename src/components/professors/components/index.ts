export * from './ButtonProfessorActive';
export * from './columns';
export * from './course';
export * from './ExcelButtonProfessors';
export * from './Header';
export * from './ProfessorDataForm';
export * from './ProfessorDialog';
