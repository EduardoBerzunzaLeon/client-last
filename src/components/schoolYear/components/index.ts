export * from './PhaseCard';
export * from './FileErrorsTable';
export * from './CurrentSubjectsForm';
export * from './FailedSubjectsForm';
export * from './ValidationPasswordForm';
export * from './SchoolYearStepForm';
