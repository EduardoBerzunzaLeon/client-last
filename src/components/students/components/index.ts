export * from './columns';
export * from './Header';
export * from './StudentDataForm';
export * from './StudentDialog';
export { ProfessorsHistoryDialog } from './professorsHistory';
