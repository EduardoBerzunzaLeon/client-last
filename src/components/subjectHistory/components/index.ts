export * from './SubjectsStudied';
export * from './CurrentSubjects';
export * from './UnstudySubjects';
export * from './columns';
export * from './PhaseDialog';
export * from './Header';
export * from './PhaseDataForm';
