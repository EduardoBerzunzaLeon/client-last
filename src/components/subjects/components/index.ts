export * from './Header';
export * from './columns';
export * from './SubjectDialog';
export * from './SubjectsList';
export * from './SubjectDetailDialog';
