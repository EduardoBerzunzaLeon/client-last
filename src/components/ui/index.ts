export * from './badge/Badge';
export * from './divider/Divider';
export * from './errorCard/ErrorCard';
export * from './excelButton/ExcelButton';
export * from './headerAdmin/HeaderAdmin';
export * from './skeleton';
export * from './skeletonDropdown/SkeletonDropdown';
export * from './slipButton/SlipButton';
export * from './spanDetail/SpanDetail';
export * from './spinner/Spinner';
export * from './spinnerRTK/SpinnerRTK';
