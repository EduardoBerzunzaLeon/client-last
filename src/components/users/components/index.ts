export * from './AdminPasswordForm';
export * from './ButtonUserBlocked';
export * from './columns';
export * from './Header';
export * from './UserDataForm';
export * from './UserDialog';
