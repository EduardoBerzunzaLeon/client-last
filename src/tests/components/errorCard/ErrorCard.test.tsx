import { shallow } from 'enzyme';

import { ErrorCard } from '../../../components/ui';

describe('<ErrorCard />', () => {
  test('should match to snapshot', () => {
    const wrapper = shallow(<ErrorCard title="Error" detail="Server Error 500" />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('img').exists()).toBeTruthy();
  });
});
