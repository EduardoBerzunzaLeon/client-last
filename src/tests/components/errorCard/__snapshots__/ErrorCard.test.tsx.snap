// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorCard /> should match to snapshot 1`] = `
<div
  className="grid"
>
  <div
    className="col-12"
  >
    <Card
      className="bg-purple-100"
      footer={null}
      header={null}
      id={null}
      style={null}
      subTitle={null}
      title="Error"
    >
      <div
        className="flex align-items-center justify-content-center"
      >
        <div
          className="fadein animation-duration-2000"
          style={
            Object {
              "maxWidth": "500px",
            }
          }
        >
          <figure>
            <img
              alt="error400"
              className="max-w-full block"
              src="/assets/images/error404.png"
            />
          </figure>
        </div>
      </div>
      <span
        className="flex align-items-center justify-content-center text-pink-900 font-bold text-xl text-justify"
      >
        ¡Ups! Nuestro perrito se comio la infomación, favor de intentarlo más tarde.
      </span>
      <span
        className="inline-block mt-2"
      >
        Error: Server Error 500
      </span>
    </Card>
  </div>
</div>
`;
